packages/react-instantsearch-native/package.json

```diff
@@ -43,7 +43,7 @@
     "react-instantsearch-core": "^5.7.0"
```

Comment:
this should become 6 I guess but we haven't released that yet

Reply:
Yep those are managed by the release script via `lerna`.